<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zhujuan</class>
 <widget class="QDialog" name="zhujuan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;AcadEref&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>欢迎使用组卷功能</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;AcadEref&quot;;
font: 12pt &quot;AcadEref&quot;;
font: 8pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string>  添加压轴题</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>试卷名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>题目数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>选择难度</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>简单</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>正常</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>困难</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;AcadEref&quot;;
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.637, y2:0.824, stop:0 rgba(85, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));

</string>
   </property>
   <property name="text">
    <string>一键生成</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(171, 255, 250);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 26pt &quot;AcadEref&quot;;
font: 20pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string>生成试卷预览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>660</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;AcadEref&quot;;
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.637, y2:0.824, stop:0 rgba(85, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>611</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.755682, stop:0 rgba(0, 158, 255, 255), stop:1 rgba(255, 255, 255, 255));
background-image: url(:/new/prefix1/V3troBZMPbzMFvzaAMl.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>checkBox</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
